doc/mediasdk-man.md

```diff
@@ -8293,6 +8330,8 @@ SDK API 1.6 added the `MFX_WRN_FILTER_SKIPPED` return status.
 
 SDK API 1.19 added `MFX_ERR_GPU_HANG` and `MFX_ERR_REALLOC_SURFACE`.
 
+SDK API 1.31 added `MFX_ERR_NONE_PARTIAL_OUTPUT`
```

Comment:
1.31 or TBD? If we miss something in this repository from API 1.31, this should be added in a separate commit.

Reply:
Is it right way to use TBD? Will it be changed to the version of next release?

Reply:
Yes, it should be changed by @onabiull

Reply:
done